src/OrbitGl/TimeGraph.cpp

```diff
@@ -521,6 +525,7 @@ void TimeGraph::PrepareBatcherAndUpdatePrimitives(PickingMode picking_mode) {
 
   batcher_.StartNewFrame();
 
+  text_renderer_static_.Init();
```

Comment:
Do we need both calls (Init and Clear)?

Reply:
Yes, `Init()` only has an effect the first time it's called. I've moved this from the constructor so we can construct `TimeGraph` in unit tests, otherwise it will fail because of missing OpenGl capabilities. We use the same pattern for the Batchers - basically everything OpenGl related is initialized the first time it's used.

Reply:
I was mostly thinking on the Clear method. Init should start with everything clear or I am wrong?